src/sentry/api/helpers/deprecation.py

```diff
@@ -55,7 +65,7 @@ def _should_be_blocked(deprecation_date: datetime, now: datetime, key: str) -> b
             brownout_duration = options.get(duration_key)
         except UnknownOption:
             logger.exception("Unrecognized deprecation duration %s", key)
-            brownout_duration = options.get("api.deprecation.brownout-duration-seconds")
+            brownout_duration = options.get("api.deprecation.brownout-duration")
```

Comment:
This now matches the default brownout-duration key name. Prior to this, if a deprecated endpoint didn't define `key` it would log an exception on every request.